lib/IDE/CodeCompletion.cpp

```diff
@@ -4575,13 +4599,14 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
         continue;
       CodeCompletionResultBuilder Builder(
           Sink, CodeCompletionResult::ResultKind::Pattern,
-          SemanticContextKind::ExpressionSpecific, {});
+          SemanticContextKind::Local, {});
```

Comment:
What's your reasoning for `Local` here?  I wonder if we need a new semantic context for this, or maybe `None`.